actors/evm/src/interpreter/system.rs

```diff
@@ -226,17 +226,21 @@ impl<'r, RT: Runtime> System<'r, RT> {
             .rt
             .send_generalized(to, method, params, value, gas_limit, send_flags)
             .map_err(|err| actor_error!(unspecified; "send syscall failed: {}", err))?;
+
+        // Don't bother reloading on abort, just return the error.
+        if !result.exit_code.is_success() {
+            return Err(ActorError::checked_with_data(
+                result.exit_code,
+                format!("failed to call {to} on method {method}"),
```

Comment:
missing the {to} param here?

Reply:
Nope. Rust format strings let you inline variables like that.

Reply:
(and they're type-checked at compile time)